[2021-05-12 10:30:17,585] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-10T00:00:00+00:00 [queued]>
[2021-05-12 10:30:17,592] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-10T00:00:00+00:00 [queued]>
[2021-05-12 10:30:17,592] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 10:30:17,592] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2021-05-12 10:30:17,592] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 10:30:17,705] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): task1> on 2021-05-10T00:00:00+00:00
[2021-05-12 10:30:17,707] {standard_task_runner.py:52} INFO - Started process 2174601 to run task
[2021-05-12 10:30:17,712] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'status_test', 'task1', '2021-05-10T00:00:00+00:00', '--job-id', '51', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_status.py', '--cfg-path', '/tmp/tmpkuexjmrc', '--error-file', '/tmp/tmpu8ekczbv']
[2021-05-12 10:30:17,713] {standard_task_runner.py:77} INFO - Job 51: Subtask task1
[2021-05-12 10:30:17,852] {logging_mixin.py:104} INFO - Running <TaskInstance: status_test.task1 2021-05-10T00:00:00+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 10:30:18,001] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=status_test
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2021-05-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-10T00:00:00+00:00
[2021-05-12 10:30:18,002] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 10:30:18,002] {bash.py:158} INFO - Running command: echo im task 1 !!!!!!!!!!!!!!!!!!!! && exit 0 || exit 1
[2021-05-12 10:30:18,010] {bash.py:169} INFO - Output:
[2021-05-12 10:30:18,011] {bash.py:173} INFO - im task 1 !!!!!!!!!!!!!!!!!!!!
[2021-05-12 10:30:18,012] {bash.py:177} INFO - Command exited with return code 0
[2021-05-12 10:30:18,130] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=status_test, task_id=task1, execution_date=20210510T000000, start_date=20210512T103017, end_date=20210512T103018
[2021-05-12 10:30:18,421] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-12 10:30:18,444] {local_task_job.py:146} INFO - Task exited with return code 0
